### DISCUSSION TITLE
AdminLTE v3.2.0

### DISCUSSION THREAD

**Discussion by REJack** [ColorlibHQ/AdminLTE#4249] [General]
__Breaking Change!__
We dropped official node v12, it might be still work but we won't give support for any node 12 bugs/errors.

__Top-Features:__
- Enhanced Dark Mode
- SVG Font Awesome Icon support
- Docker dev-preset
- Reworked Dark/Light colors
- Custom Scrollbar style (Non-Plugin-Scrollbar)

__Fixes:__
- various fixes in IFrame plugin
- fix link encoding in SidebarSearch plugin
- fix division warning while sass building
- fix disabled state for default & outline-light button
- fix none-visible background colors while printing
- various dark-mode fixes

__Changes:__
- add load error fallback for CardRefresh plugin
- add collapsed-done event for ControlSidebar plugin & PushMenu plugin
- add animationSpeed to ControlSidebar plugin & PushMenu plugin
- add target selection for ControlSidebar plugin
- add element fallback for ExpandableTable plugin
- add allowReload & autoDarkMode to IFrame plugin
- add iframe-dark example

Thanks for your awesome contribution @liljack, @danny007in, @curiousteam, @equada, @Shidersz, @gurvirlochab, @ZhangChengLin, @mostafahesham-dev, @smtbos, @leonardxfce, @anilloutombam, @13324, @SmileYzn, @hkvstore, @rikuson, @kreic51, @forxer & @mahmoudalsaman.

For the complete changelog [look here](https://github.com/ColorlibHQ/AdminLTE/compare/v3.1.0...v3.2.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/ColorlibHQ/AdminLTE/releases/tag/v3.2.0'>AdminLTE v3.2.0</a>.</em>

**Comment by REJack** [comment]
@danny007in Could you create a summary of done v4 stuff and todos 😄 also did you added darkmode to it already? If yes, I will implement the skin system first to keep it small. (not like v3 🤣)

**Reply by danny007in** [reply]
todo https://github.com/ColorlibHQ/AdminLTE/projects/13

skin system will be like --sidear-color-primary (css variable) 
please check code

**Reply by spizzo14** [reply]
@danny007in @REJack hey guys, is adminlte on bootstrap 5 yet? Just curious i know i emailed a few times, but i'm also willing to pay a developer to help us upgrade

**Reply by danny007in** [reply]
> @danny007in @REJack hey guys, is adminlte on bootstrap 5 yet? Just curious i know i emailed a few times, but i'm also willing to pay a developer to help us upgrade

There is sponsor button for repo and user 👍🏼

**Reply by spizzo14** [reply]
We can absolutely do that. Do you have an idea when it will be updated? Can I pay more to get it done sooner?

**Reply by spizzo14** [reply]
@danny007in hopefully you got my sponsor ok! send me a DM if you have any opportunities to do more work

**Comment by datnttg** [comment]
There is a issue with Select2 plugin. The record is not autofocus in search field.

**Reply by REJack** [reply]
that you should report to the select2 repo. We don't maintaine any of the plugins.

**Reply by forxer** [reply]
There is already an issue about this, and a workaround: https://github.com/select2/select2/issues/5993#issuecomment-800927000

**Comment by ghost** [comment]
Hi, how I configure collapse sidebar by default, when I refresh the page?

**Reply by dfsmania** [reply]
I think you need to add `sidebar-collapse` class to the set of body classes, example:

```html
<body class="sidebar-collapse ...">
</body>
```

**Comment by aamsoh** [comment]
Hi, 

I have integrated Admin LTE v3.2 with React JS, everything is working fine except dropdown menu is not opening and side nav bar do not show scroll bar . please support. 

Best Regards, 

Aamir
<img width="960" alt="Admin LTE" src="https://user-images.githubusercontent.com/32142891/174228487-6400494f-1ad3-4fac-8dc3-646fff3415a4.png">

**Reply by LunaTheGuardian** [reply]
> Hi,
> 
> I have integrated Admin LTE v3.2 with React JS, everything is working fine except dropdown menu is not opening and side nav bar do not show scroll bar . please support.
> 
> Best Regards,
> 
> Aamir <img alt="Admin LTE" width="960" src="https://user-images.githubusercontent.com/32142891/174228487-6400494f-1ad3-4fac-8dc3-646fff3415a4.png">
--------------------------------------------------------------------------------------------------------------------------------------------------------
Jquery Conflict causes these issues brother. are you included extra jquery

**Comment by aamsoh** [comment]
[image: image.png]

These are the jquery i have included which was mentioned in download
itself.

Best Regards,

Aamir

**Reply by LunaTheGuardian** [reply]
missing image brah

**Comment by aamsoh** [comment]
Please check now I have attached.

BR,

**Comment by aamsoh** [comment]
Hi,

Please can I have an update ?

BR,

Aamir

**Comment by sadithsanchez** [comment]
Estoy usando esta ultima version AdminLTE-3.2.0.zip, y el modo dark es interesante aun estoy indagando pero ya tengo unas sugerencias de mejora

**Comment by haipt130122** [comment]
Hello, admin can record a video tutorial on how to fix css, js with grunt, I can't
https://adminlte.io/blog/customizing-and-downsizing-adminlte-to-match-your-businsess/

**Comment by mike2391** [comment]
![image](https://user-images.githubusercontent.com/85857884/224321479-29058f22-d944-46f9-962e-297243c8f2c6.png)
something wrong with my code but i can't find it. any master, could you help me to solve it?

**Reply by wellyriyadipratama** [reply]
i had the same problem, have you found the solution.

**Comment by isyourjr** [comment]
Hi, for this source code can I use for my own shop website design? Is free source code right to publish public , However,  I will modify the code as well.

**Comment by filipgorczynski** [comment]
Hello,
I couldn't find any suitable place to ask that.
Does something changed lastly in the space of default branches master and/or v3? I had a project that pulls AdminLTE as submodule and by default it was set to v3 by default, but right now pulling it gives different code (v4 releases). Anything changed?

**Reply by danny007in** [reply]
sorry for delay, i am in tight schedule due to my academics, if free after April 2024

**Comment by tasnim-omrani** [comment]
Hello, i want to integrate this template in my symfony project. What are the steps to do so? Thank you for your help.

**Comment by antena-kipas** [comment]
Is there any CDN link for this version?
